parser-library/parse.cpp

```diff
@@ -136,7 +137,7 @@ static const char *pe_err_str[] = {"None",
                                    "Unable to stat",
                                    "Bad magic"};
 
-int GetPEErr() {
+std::uint32_t GetPEErr() {
```

Comment:
This was often assigned to std::uint32_t variables, so I changed the return type.